torch/_export/passes/add_runtime_assertions_for_constraints_pass.py

```diff
@@ -93,7 +93,7 @@ def add_assertions(val):
             call_backs: List[Callable] = []
             messages: List[str] = []
             if isinstance(val, (torch.SymInt, torch.SymFloat, torch.SymBool)):
-                symbol = val.node._expr
```

Comment:
how did this blow up / are there any tests this fixes?

Reply:
The export tests see the old u2 rather than the post replacement u0 when they access this directly. So... yes :)